### PR TITLE
[Crowdstrike Falcon] Update CrowdStrikeFalcon.py - Fixed the function to retrieve correct ioa rules correctly.

### PR DESCRIPTION
**Pull request by zeekforit**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixed the function to retrieve correct ioa rules correctly. This has been tested on our environment.
This is related to the recent changes from these pull request
https://github.com/demisto/content/pull/31992
https://github.com/demisto/content/pull/32124

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py [hunk 1]
```diff
@@ -1293,10 +1293,15 @@ def get_ioarules(rule_ids: list[str]) -> dict:
         :param rule_ids: IDs of the requested ioa rule.
         :return: Response json of the get ioa rule entities endpoint (ioa rule objects)
     """
+    print("RuleID:")
+    print(rule_ids)
+
+    params = { 'ids': rule_ids }
```
Comment (yucohen) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:1299]:
```suggestion
    params = {'ids': rule_ids}
```

Comment (yucohen) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:1299]: We do not use `print` in integrations :)

**Review by yucohen — CHANGES_REQUESTED** [review]
Thanks again for your contribution! 
Please also add a Release notes file and bump the version

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py (hunk 1 above)
Comment (zeekforit) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py]: Forgot to delete those haha.

**Review by zeekforit — COMMENTED** [review]
Done

**Review by yucohen — APPROVED** [review]

**Pull request merged**